### DISCUSSION TITLE
number of epochs in meta.json?

### DISCUSSION THREAD

**Discussion by DSLituiev** [explosion/spaCy#10886] [New Features & Project Ideas]
It would be great to have number of epochs in meta.json or any other file documenting training parameters.
I know there is `max_epochs` in config.cfg, but that does not help when looking in `model-best`.

Maybe there is one that I am not aware of?

My use case: I've trained models in cross-validation, and now I'd like to re-train a model on whole dataset. I need to know the optimal epoch number, but now the only way seems to run it again and look into the prompt output.

**Comment by polm** [comment]
No, that information isn't stored anywhere at present. Right now the best thing is to have your logger manage it.

Thanks for suggesting it though, we'll discuss it within the team.
